transaction/std/src/input_materials.rs

```diff
@@ -66,7 +64,7 @@ impl InputMaterials {
 // Helper which converts from InputMaterials (TransactionBuilder type) to
 // InputRing (rct_bulletproofs type)
 impl TryFrom<InputMaterials> for InputRing {
-    type Error = TxOutConversionError;
+    type Error = TxBuilderError;
```

Comment:
Whats the reason for this change?

Reply:
:+1: lets please not use a more expansive error type if not necessary